recipes/setupmeta/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "setupmeta" %}
+{% set version = "2.7.8" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 48b8dbbdd8b85a8cc76b9bfe557d538384cfe1de1ef5112e54400f3bed49456c
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+    - setuptools
+
+test:
+  imports:
+    - setupmeta
+
+about:
+  home: "https://github.com/zsimic/setupmeta"
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE (inside source tarball)
```

Comment:
Sorry if I wasn't clear earlier - please leave `license_file: LICENSE` here. The path to the license file, LICENSE, is relative to the source directory, which is the tarball, where the license is.

Reply:
Also, please remove `(inside source tarball)` since that field should only contain the name of the license file.